src/p2p/node.ts

```diff
@@ -41,10 +42,10 @@ export class LodestarNode extends LibP2p {
   public static createNode(callback): LodestarNode {
     let node: LodestarNode;
 
-    const id = await promisify(PeerId.create)({ bits: 1024});
-    const peerInfo = await promisify(PeerInfo.create)(id);
+    const id = promisify(PeerId.create)({bits: 1024});
+    const peerInfo = promisify(PeerInfo.create)(id);
     peerInfo.multiaddrs.add('ip4/0.0.0.0/tcp/9000');
-    const node = new LodestarNode({
+    node = new LodestarNode({
```

Comment:
You can remove the `let` on line 43 and replace line 48 with the assignment
`const node: LodestarNode = new LodestarNode({ ... })`